src/main.ts

```diff
@@ -198,7 +198,8 @@ export class CustomSonosCard extends LitElement {
         --sonos-int-artist-album-text-color: var(--sonos-artist-album-text-color, var(--primary-text-color));
         --sonos-int-accent-color: var(--sonos-accent-color, var(--accent-color));
         --sonos-int-title-color: var(--sonos-title-color, var(--card-background-color));
-        --sonos-int-border-radius: 0.25rem;
+        --sonos-int-border-radius: var(--sonos-border-radius, 0.25rem);
+        --sonos-int-border-width: var(--sonos-border-width, 8px);
```

Comment:
I'd like to use rem unit, is that possible?

Reply:
Sure, I only used the 8px because it was there already. I only use rem where you used rem and only px where you used px. Be free to change anything I do I don't mind.